.gitignore

```diff
@@ -12,3 +12,4 @@
 !/log/.keep
 /tmp
 .byebug_history
+.idea
```

Comment:
This a Rubymine internal version control thing. It helps with telling other RM clients what the structure of the 'project' is. [Their docs are better at explaining](https://www.jetbrains.com/ruby/help/project.html)